src/WebJobs.Script/Workers/MessageExtensions/ScriptInvocationContextExtensions.cs

```diff
@@ -173,6 +173,14 @@ internal static object GetHttpScriptInvocationContextValue(object inputValue, Da
                 }
                 return Convert.ToBase64String(byteArray);
             }
+            if (inputValue is DateTime)
+            {
+                return DateTime.Parse(inputValue.ToString());
+            }
+            if (inputValue is DateTimeOffset)
+            {
+                return DateTimeOffset.Parse(inputValue.ToString());
+            }
```

Comment:
This would be a breaking change, right ? Should the release notes mention that ?

Reply:
yes! updated release notes.

Reply:
Sorry, what I meant was, release notes explicitly mentioning **[BreakingChange]** or something like that.

Reply:
I have grouped breaking changes for custom handler under one bullet - https://github.com/Azure/azure-functions-host/blob/pgopa/fixDateSerialization/release_notes.md - can you please check the format?

Reply:
Ah, I missed that. Thanks for clarifying ! Yeah, format looks good